door-open-service/node-backend/app.js

```diff
@@ -13,7 +13,7 @@ const _ = require('underscore');
 const BellServer = require('./bellServer');
 const backend = require('./backend');
 
-const CHANNEL = '<CHANNEL_NAME>';
+const CHANNEL = { name: '<CHANNEL_NAME>', private: true };
```

Comment:
Sure! A big refactoring with more than just extracting the variables is scheduled :) But I'll open a dedicated PR just for that; right now the immediate goal is to implement the necessary to add quick support for more doors.